deploy/helms/legion-core/templates/liara/liara-secrets.yaml

```diff
@@ -0,0 +1,32 @@
+{{ if .Values.liara.enabled -}}

```

Comment:
Why do we need to have urls in configMap?

Reply:
Because we have no another way now as I know.